### ISSUE TITLE
Need Section508 Level Support

### ISSUE THREAD

**Issue by kjschulz** [yargalot/AccessSniff#21]
This is a great tool but in order to use it on most of my government-led projects, I need to be able to specify `accessibilityLevel: 'Section508'` instead of only `WCAG2A[A][A]` as currently that is the minimum level our applications must be compliant with in order to ship. I don't disagree that we should be working to support WCAG2A at least but that is another battle. Since this option is already baked in to HTML_CodeSniffer why is this not extended to AccessSniff?

**Comment by yargalot** [comment]
Hey, totally agree. Im overseas at the moment so will integrate this in and push it out on the 28th :)
